browser/src/components/Pill/Pill.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
+
+class Pill extends Component {
+  render() {
+    return <div>this is a Pill component</div>;
+  }
+}
```

Comment:
Hey, this component doesn't appear to have or need any state - could we turn it into a function component?